libs/CCAppCommon/devices/3dConnexion/cc3DMouseManager.cpp

```diff
@@ -330,7 +332,7 @@ void cc3DMouseManager::on3DMouseReleased()
 		win->redraw();
 	}
 }
-
+// sets up the menu for 3D Mouse
```

Comment:
documentation the function in the header file should be sufficient? (moreover most the methods are not documented this way in cpp files, and it's hard to keep the consistency between the header and the cpp file)